components/Settings/ClubTab.js

```diff
@@ -0,0 +1,87 @@
+import { DARK_GRAY } from '../../constants/colors'
+import { CLUB_ROUTE } from '../../constants/routes'
+import s from 'styled-components'
+import Toggle from './Toggle'
+import { Icon } from '../common'
+import ReactTooltip from 'react-tooltip'
+
+const Table = s.table`
```

Comment:
Should we put this along w/ the other shared components? Would be nice to use the same table styles everywhere

Reply:
i'm going to put this off, we don't have tables elsewhere and passing in the icon and tooltip will be annoyingggggg